pkg/utils/environment_variable_test.go

```diff
@@ -0,0 +1,105 @@
+package utils
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+const (
+	EnvVarName             = "SOME_ENV_VAR"
+	IntEnvVarValueStr      = "10"
+	IntEnvVarValue         = 10
+	DurationEnvVarValueStr = "10m0s"
+	DurationEnvVarValue    = 10 * time.Minute
+	InvalidValueStr        = "SOME_ENV_VAR_VALUE"
+)
+
+// This mock function always returns the given default value
+func MockEnvVarValueOrDefaultFunc(envVarName string, defaultValue string) string {
+	return defaultValue
+}
+
+func TestEnvironmentVariable(t *testing.T) {
+	Convey("IntFromEnvOrDefault", t, func() {
```

Comment:
I think we're deprecating the use of `Convey`? But personally I'm fine with it... it tells the story of what you're testing.

Reply:
@diverdane what are we moving to if not `Convey`